components/StyledUpdate.js

```diff
@@ -241,13 +245,19 @@ class StyledUpdate extends Component {
   }
 
   renderFullContent() {
-    const { update, collective, isReloadingData } = this.props;
+    const { update, collective, isReloadingData, reactions, LoggedInUser } = this.props;
 
     return (
       <Container css={{ wordBreak: 'break-word' }} pl={[0, 60]} maxWidth={676}>
         <StyledHr mt={3} mb={4} borderColor="black.100" />
-        {update.html ? (
-          <HTMLContent content={update.html} />
+        {update.html && update.publishedAt && LoggedInUser ? (
```

Comment:
This is not the right place to add the `LoggedInUser` check: we want everyone to see existing reactions. Only the `EmojiReactionPicker` should be hidden to logged-out users.

Reply:
Ops, you are right. We need to add this to the `EmojiReactionPicker` only. Did that. 😄

Reply:
@SudharakaP You forgot to remove the `&& LoggedInUser`. 

Also the `&& update.publishedAt check` is a problem as we're not going to see the content of drafts anymore. Try creating a new update, click on "Preview" and you'll see that the content is missing.

Reply:
@Betree : Sorry my mistake. I've removed the `LoggedInUser` condition and moved the `update.publishedAt` check as well. Now it should be good. The `publishedAt` should only hide the emojis when it's on draft mode. 😄